Input/input.css

```diff
@@ -0,0 +1,107 @@
+:root {
+  --gray: #3e4152;
+  --blue: #084298;
+  --red: #dc3545;
+  --text-color: #007fff;
+  --bg-code: #fafafa;
+}
+body {
+  font-family: "Poppins", sans-serif;
+  margin: 2rem;
+  color: var(--gray);
+}
+h1 {
+  font-size: 3rem;
+}
+
+h1,
+h2,
+p {
+  margin: 0.625rem 0;
+  line-height: 1.2;
+}
+
+h2 {
+  font-size: 2rem;
+}
+p {
+  font-size: 1.2rem;
+  padding: 0.8rem;
+}
```

Comment:
If these styles are not specific to this component then put them in one common CSS file.

Reply:
Rest looks good to me.